### PR TITLE
[Project] Replace project URLs with more user friendly URLs

### LINKED ISSUE #590: [Project] Replace project URLs with more user friendly URLs
`/p` -> `/project`
`/p/:new` -> `/project/new`

### PR DESCRIPTION
**Pull request by DakshSaini**
Closes #590

### CODE REVIEW

**Review by gino-m — CHANGES_REQUESTED** [review]
Nice work! Would it be possible to roll a small refactor into this PR? Namely, it seems "project" and "new" are sprinkled across our codebase. As a quick fix, can we always refer to constants in the router instead? In a follow-up PR we can also replace calls to `this.router.navigate()` with methods in our `NavigationService`. Wdyt?

**Comment by DakshSaini** [comment]
> Nice work! Would it be possible to roll a small refactor into this PR? Namely, it seems "project" and "new" are sprinkled across our codebase. As a quick fix, can we always refer to constants in the router instead? In a follow-up PR we can also replace calls to `this.router.navigate()` with methods in our `NavigationService`. Wdyt?

Sounds good, changes made. I'll be sure to create a follow-up PR.

File: web-ng/src/app/services/router/router.service.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ export class NavigationService {
   private static readonly OBSERVATION_ID_FRAGMENT_PARAM = 'o';
   static readonly LAYER_ID_NEW = 'new';
   static readonly OBSERVATION_ID_NEW = 'new';
+  static readonly PROJECT_ID_NEW = 'new';
+  static readonly PROJECT_ID = 'projectId';
+  static readonly DEFAULT_PROJECT_ROUTE = 'project';
```
Comment (gino-m) [inline web-ng/src/app/services/router/router.service.ts:38]: Why is this the "default" project route? Shouldn't it be something like the "project URL base path segment"? (`PROJECT_SEGMENT`?)

File: web-ng/src/app/services/router/router.service.ts [hunk 2]
```diff
@@ -33,6 +33,10 @@ export class NavigationService {
...
+  static readonly SIGN_IN_ROUTE = 'signin';
```
Comment (gino-m) [inline web-ng/src/app/services/router/router.service.ts:39]: Same here - this isn't a complete route, just a segment. Please rename accordingly.

**Review by gino-m — APPROVED** [review]
LGTM! Filed #605 for followup refactor. Thanks!!!

**Pull request merged**